builtin/types/config/config.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+import "github.com/loomnetwork/go-loom/types/types.proto";
+
+enum ReceiptStorage {
+    CHAIN = 0;
+    LEVELDB = 1;
+}
+
+message ConfigInitRequest {
+    Address oracle = 1;
+    repeated KeyValue settings = 2;
+}
+
+message Key {
```

Comment:
not sure this needs its own message

Reply:
unless this is supposed to be GetKey ? or GetSetting?

Reply:
It's the input parameter to the Get method, `(c *Config) Get(ctx contractpb.StaticContext, key ctypes.Key ) (*ctypes.Value, error)`

Reply:
Maybe should be changed to RetrieveSetting or similar.

Reply:
GetSetting